### DISCUSSION TITLE
What is `snapshots` in lockfile version 9?

### DISCUSSION THREAD

**Discussion by remorses** [pnpm/pnpm#8035] [Q&A] [answered]
I am debugging why i have 2 different versions of `next` installed (because of some peer dependency getting duplicated), i only have 1 version if next in the `packages` list and another one in the `snaphosts` list. What is snapshots? how is it different than packages?

**Comment by cdaringe** [comment, marked as answer]
https://github.com/pnpm/pnpm/issues/7685 talks about it.

it seems like TLDR snapshots represents all of the logical packages present in the stack
